pkg/operator/controllers/guardrails/policies/gktemplates-src/aro-deny-regularuser-apigroups/aro-deny-regularuser-apigroups.tmpl

```diff
@@ -0,0 +1,24 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: arodenyregularuserapigroups
```

Comment:
For consistency this name should probably be aro-deny- etc.

Reply:
I think all the Constraint Template follow the same name patter without the `-` in between.